### DISCUSSION TITLE
Setting up MATLAB LSP

### DISCUSSION THREAD

**Discussion by Lilweavs** [helix-editor/helix#7653] [General]
I am trying to get the official MATLAB LSP setup (https://github.com/mathworks/MATLAB-language-server). Though it officially only supports VScode, it appears to work with Neovim (https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#matlab_ls), so I have been trying to get it to work in Helix.

I have zero experience with Typescript and Javascript, and I am struggling to get "matlab-language-server" as a runnable command. I have it installed via `npm install -g` and I can see the package show up via `npm list -g`. Has anyone tried to get this LSP working with Helix, or has any pointers on how to properly compile this.

**Comment by NonadonSon** [comment]
Therefore,are you succeed? Can you tell me how to configure matlab-ls?

**Comment by SubtleMuffin** [comment]
Just created an issue and recorded my attempts so far: https://github.com/helix-editor/helix/issues/8947.

**Comment by SubtleMuffin** [comment]
Now the problem is resolved. Please see the discussion in #8947.
